### ISSUE TITLE
relative application root produces a basePath with a trailing slash that breaks links

### ISSUE THREAD

**Issue by Mykro** [stoicflame/enunciate#192]
I'm having problems with the "Try it now!" links in the Swagger UI.

My server initialisation:
```
    server = GrizzlyHttpServerFactory.createHttpServer( URI.create("http://localhost:9999/api"), resourceConfig );
    StaticHttpHandler staticHttpHandler = new StaticHttpHandler( "www");
    server.getServerConfiguration().addHttpHandler(staticHttpHandler, "/");
```
This results in the following urls:
    http://localhost:9999/api 
    http://localhost:9999/apidocs <-- enunciate docs are deployed here

My enunciate config:
````
    <enunciate xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
        <application root="/api" />
    </enunciate>
````

**Problem**
The above config generates apidocs/ui/swagger.json that contains: `"basePath" : "/api/", `

Note enunciate appends a slash to the application root.  In the Swagger UI the "Try it out!" button produces `curl -X GET --header "Accept: text/html" "http://localhost:9999/api//project" `

Note that Swagger has appended yet another slash - this URL fails to resolve.

At the moment the only workaround is to manually edit swagger.json to remove the trailing slash from basePath before each deployment.

**Comment by stoicflame** [comment]
Thanks for the report.

**Comment by stoicflame** [comment]
Fixed at bca5dfd.

A current workaround is to specify a different swagger basePath:

```xml
<enunciate>
  <modules>
    <swagger basePath="/api"/>
```

**Comment by stoicflame** [comment]
[Enunciate 2.1 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.1.0)
